das/daser.go

```diff
@@ -63,8 +88,10 @@ func NewDASer(
 	getter header.Getter,
 	dstore datastore.Datastore,
 	bcast fraud.Broadcaster,
+	options ...Option,
 ) *DASer {
 	d := &DASer{
+		params:         dasingParams{},
```

Comment:
Could we please start from some default config instead of zero-value here so if there some options are missing (they are optional, right?) we don't end up with 0 in some config fields. It could make DASer broken otherwise.

Reply:
Exactly, I hear you. I thought about this.
Noted.